### PR TITLE
Add tx building for cosmos and kujira

### PR DESCRIPTION
**Pull request by alexisshleverx**

### CODE REVIEW

File: packages/toolboxes/cosmos/src/util.ts [hunk 1]
```diff
@@ -27,6 +28,17 @@ export const DEFAULT_COSMOS_FEE_MAINNET = {
   gas: "200000",
 };
 
+export const getDefaultChainFee = (chain: CosmosChain) => {
+  switch (chain) {
+    case Chain.Maya:
+      return { amount: [], gas: "10000000000" };
+    case Chain.THORChain:
+      return { amount: [], gas: "500000000" };
+    default:
```
Comment (towanTG) [inline packages/toolboxes/cosmos/src/util.ts:45]: kujira case missing here I think

File: packages/toolboxes/cosmos/src/toolbox/BaseCosmosToolbox.ts [hunk 2]
```diff
@@ -100,6 +112,49 @@ export const BaseCosmosToolbox = ({
...
+          amount: assetValue.getBaseValue("string"),
+          denom: getDenomWithChain(assetValue),
+        },
+      ],
+    };
+    return {
+      memo,
+      accountNumber: accountOnChain.accountNumber,
+      sequence: accountOnChain.sequence,
+      chainId: ChainId.THORChain,
```
Comment (towanTG) [inline packages/toolboxes/cosmos/src/toolbox/BaseCosmosToolbox.ts]: this needs to be set based on chain

File: packages/toolboxes/cosmos/src/util.ts [hunk 3]
```diff
@@ -50,6 +62,28 @@ export const getDenom = (symbol: string, isThorchain = false) => {
...
+  }
+
+  if (chain === Chain.THORChain) {
+    return (
+      symbol.toUpperCase() !== "RUNE" ? symbol : `${Chain.THORChain}.${symbol}`
+    ).toUpperCase();
+  }
+
+  if (chain === Chain.Kujira) {
+    return (symbol.toUpperCase() !== "KUJI" ? symbol : `${Chain.Kujira}.${symbol}`).toUpperCase();
```
Comment (towanTG) [inline packages/toolboxes/cosmos/src/util.ts]: should return 'ukuji' imo

Comment (towanTG) [reply packages/toolboxes/cosmos/src/util.ts]: ah I will have to read through the cosmos stuff again - I dont think we need this method for other chains but MAYA + THOR

File: packages/toolboxes/cosmos/src/util.ts (hunk 1 above)
Comment (alexisshleverx) [reply packages/toolboxes/cosmos/src/util.ts:45]: May you hint me with correct gas value for kujira?

**Pull request closed without merging**